### DISCUSSION TITLE
BTCPay 2.0 migration crash - report problem here

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#6296] [General]
Hey,

In case you've updated to 2.0 and experiencing issues with migrations ,hang tight! We will prioritize your reports here. Please leave out any relevant information while filling in a report, such as version you've upgraded from, deployment method and most importantly server logs.

We don't expect crashes to happen, but if they do don't worry, we'll do our best to fix things asap!

**Comment by iBobik** [comment]
Hello,
I experience strange error after upgrade - BTCPay does not send responses (so nginx fails with 504 Gateway timeout).

Discussing it here: https://chat.btcpayserver.org/btcpayserver/pl/ddndohjxe7g4bef9zug55q8uzo

**Comment by idrone3d** [comment]
Hi team,

I'm currently on version 2.0.7 of BTCPay Server and attempted to update to version 2.1 using the usual commands:
![Screenshot_1](https://github.com/user-attachments/assets/98cc36d6-959a-4a58-ab05-e47f81d9950e)

cd /root/BTCPayServer/btcpayserver-docker
git fetch -a
git checkout 2.1
./btcpay-update.sh
However, the update script runs without any visible output or errors, and the version does not change. I’ve tried this multiple times, even re-checking permissions and the script itself.

Can you confirm if anything has changed in the upgrade process from 2.0.x to 2.1? Is there a new method or flag required?

Thanks in advance for the help, and let me know if you need logs or further details from my instance.

Best regards,
David

**Reply by pavlenex** [reply]
Just do btcpay-update.sh form 2.0.7

**Reply by Deezy33-creator** [reply]
> Just do btcpay-update.sh form 2.0.7

@rockstardev
